modules/kit-elements-defaults/import-export/import-export.php

```diff
@@ -11,6 +11,8 @@
 }
 
 class Import_Export {
+	const FILE_NAME = 'kit-elements-defaults';
```

Comment:
It's the same name as the module. I meant that you can use the module's constant instead.

Reply:
I do not want to mix between those consts, they are same but they are not depends on each other